### PR TITLE
Update README and timestamp

### PR DESCRIPTION
**Pull request by LionetChen**
Add installation instructions and other info including a details section

### CODE REVIEW

**Comment by nalachen2020** [comment]
The master has got nothing. Just resolve the conflict by using code from your branch.

**Comment by LionetChen** [comment]
Correct way to do it is to merge master into the branch and solve the conflict. Then push origin

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
 # The Best Example Ever
 Which will be a part of the best commit ever
 
+## Installation
+Just browser it with Chrome
+
+## Other Info
+
```
Comment (nalachen2020) [inline README.md:8]: This empty line is not necessary

File: README.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+Master branch is protected since 2210 hours on 2022-08-14
+
+<details>
+
+### More details
+More to come
+
```
Comment (nalachen2020) [inline README.md:15]: I'm not sure about this empty line as well

File: README.md (hunk 1 above)
Comment (nalachen2020) [inline README.md:8]:
```suggestion

```
Just delete it maybe? @LionetChen

File: README.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
 # The Best Example Ever
 Which will be a part of the best commit ever
```
Comment (nalachen2020) [inline README.md:2]:
```suggestion
Which will be a part of a very good commit
```
Dial it down a bit?

**Review by nalachen2020 — COMMENTED** [review]
Please see if the things I suggested is applicable

File: README.md (hunk 1 above)
Comment (LionetChen) [reply README.md]: Agree

File: README.md [hunk 4]
```diff
@@ -1,3 +1,16 @@
...
 
+## Installation
+Just browser it with Chrome
```
Comment (nalachen2020) [inline README.md:5]: I like these 2 lines

Comment (LionetChen) [reply README.md:5]: Glad you do

File: README.md (hunk 2 above)
Comment (LionetChen) [reply README.md:14]: Leave it for now

**Review by LionetChen — COMMENTED** [review]
See if you like it now?

**Review by nalachen2020 — APPROVED** [review]
I would approve.

**Pull request merged**